### PR TITLE
add pair 'd': '0' to digit lookalikes

### PR DESCRIPTION
**Pull request by Numerlor**

### CODE REVIEW

File: ratlib/autocorrect.py [hunk 1]
```diff
@@ -15,15 +15,16 @@
 
 class CorrectionResult:
     # Table of lookalikes
-    lookalikes = {'L': dict(zip("01258", "oizsb"))}
-    lookalikes['D'] = dict((v, k) for k, v in lookalikes['L'].items())
+    digits = "001258"
+    letters = "doizsb"
+    lookalikes = {'L': dict(zip(digits, letters)), 'D': dict(zip(letters, digits))}
```
Comment (theunkn0wn1) [inline ratlib/autocorrect.py:20]: if we are going to change these lines around, we may as well use dictionary comprehensions.

File: ratlib/autocorrect.py [hunk 2]
```diff
@@ -15,15 +15,16 @@
...
 
     # Allowed characters for each pattern.  Matches regex syntax within a [group]
     allowed = {'L': 'a-z', 'D': '0-9'}
 
     # Create the regular expression we use to match systems
     pattern = (r'\w+\s+(?P<l>LL-L\s+L)(?P<d>D+(?:-D+)?'
-               r')\b')  #  L and D will be replaced from lookalikes and alowed.
+               r')\b')  # L and D will be replaced from lookalikes and allowed.
```
Comment (theunkn0wn1) [inline ratlib/autocorrect.py:27]: please undo this modification, there is by PEP8 supposed to be two spaces after `#` for inline comments.

File: ratlib/autocorrect.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 """
 System name autocorrection
 
-Copyright (c) 2017 The Fuel Rats Mischief, 
+Copyright (c) 2017 The Fuel Rats Mischief,
```
Comment (theunkn0wn1) [inline ratlib/autocorrect.py:4]: unnecessary modification.

**Review by theunkn0wn1 — CHANGES_REQUESTED** [review]
What use case brought this up? Does this refer to a MHD ticket (doesn't need one, im just wondering if it does)? can we see some demonstrations on the functionality you are adding (tests) ?

also, some structural things in this pr substance could use some more work, while we are modifying this part.

File: ratlib/autocorrect.py (hunk 2 above)
Comment (Numerlor) [reply ratlib/autocorrect.py:27]: Inline comments should start with '# ' according to PEP8 
https://www.python.org/dev/peps/pep-0008/#inline-comments

Comment (theunkn0wn1) [reply ratlib/autocorrect.py:27]: it appears i misread the part `Inline comments should be separated by at least two spaces from the statement. `

in either case, this is an out-of-scope change.

Comment (Marenthyu) [reply ratlib/autocorrect.py:27]: We're not discussing PEP8 stuff for this repo, it's getting replaced anyway.

File: ratlib/autocorrect.py (hunk 3 above)
Comment (kenneaal) [reply ratlib/autocorrect.py:4]: This is mecha2. I think you may be applying project standards from Mecha3 to a rather simple hotfix for Marenthyu's project here...

Comment (Marenthyu) [reply ratlib/autocorrect.py:4]: It's whatever. Yes, unnecessary, but who cares, really?

**Review by Marenthyu — APPROVED** [review]
D and 0 look sufficiently similar in E:D's font, so this is a reasonable change.

**Pull request merged**